CMakeLists.txt

```diff
@@ -414,7 +410,15 @@ if(ENABLE_HEADLESS)
 	add_definitions(-DUSE_HEADLESS)
 endif()
 
-add_definitions(-D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE)
+# Set file offset size to 64 bits.
+#
+# On modern Unixes, this is typically already the case. The lone exception is
+# glibc, which may default to 32 bits. glibc allows this to be configured
+# by setting _FILE_OFFSET_BITS.
+if(NOT CMAKE_SYSTEM_NAME MATCHES "Windows")
+	add_definitions(-D_FILE_OFFSET_BITS=64)
```

Comment:
I guess that matters only on a few systems that have it optional, most libc now actually default to 64bit offsets and don't require those.